core/server/common/src/main/java/alluxio/conf/ServerConfiguration.java

```diff
@@ -150,7 +149,7 @@ public static String get(PropertyKey key, ConfigurationValueOptions options) {
    * @param defaultValue the value to return if no value is set for the specified key
    * @return the value
    */
-  public static String getOrDefault(PropertyKey key, String defaultValue) {
+  public static Object getOrDefault(PropertyKey key, String defaultValue) {
```

Comment:
Any chance to make this one as public static <T> T getOrDefault(PropertyKey key, T defaultValue)?
Not sure if it's feasible or not....